src/Lesson2/Lesson2.java

```diff
@@ -0,0 +1,224 @@
+
+                            switch (choice) {
+                                case '1':
+                                    Scanner console = new Scanner(System.in);
+                                    System.out.println("Введите год");
+                                    System.out.println(year(console.nextInt()));
+                                    break;
+                                case '2':
+                                    if (random.nextBoolean()) {
+                                        System.out.println(year(year[random.nextInt(year.length)]));
+                                    } else {
+                                        System.out.println(year(random.nextInt(3000)));
+                                    }
+                                    System.out.println();
+                                    break;
+                            }
+                        }
+                    case '6':
+                        painting();
+                        System.out.println();
+                        break;
+                }
+            }
+        }
+    }
+
+/* Задание 1. Написать метод, принимающий на вход два целых числа
+и проверяющий, что их сумма лежит в пределах от 10 до 20 (включительно),
+если да – вернуть true, в противном случае – false. */
+    private static boolean summ(int a, int b){
+        System.out.println("Задание 1\na = " + a + "\nb = " + b + "\nsum = " + (a + b));
+
+        int c = a + b;
+        return c >= 10 && c <= 20;
+    }
+
+/* Задание 2. Написать метод, которому в качестве параметра передается целое число, метод должен
+напечатать в консоль, положительное ли число передали или отрицательное. Замечание: ноль
+считаем положительным числом.*/
+    private static String strSignOfNum(int a) {
+        System.out.print("\nЗадание 2\nЧисло: " + a);
+
+        if (a >= 0) {
+           return " -- положительное";
+        }else return " -- отрицательное";
+    }
+/* Задание 3. Написать метод, которому в качестве параметра передается целое число. Метод должен
+вернуть true, если число отрицательное, и вернуть false если положительное.*/
+    private static boolean boolSignOfNum(int a){
+        System.out.println("Задание 3\na = " + a);
+
+        return a < 0;
+    }
+/* Задание 4. Написать метод, которому в качестве аргументов передается строка и число, метод должен
+отпечатать в консоль указанную строку, указанное количество раз;*/
+    private static void printStringN(String m, int n){
+
+        for (int i = 0; i < n; i++) {
+            System.out.println(m);
+        }
+    }
+/* Задание 5. * Написать метод, который определяет, является ли год високосным, и возвращает boolean
+(високосный - true, не високосный - false). Каждый 4-й год является високосным, кроме каждого
+100-го, при этом каждый 400-й – високосный.*/
+    private static boolean year(int year){
+        System.out.println(year + "г.");
+        if (year % 4 == 0 && year % 100 == 0 && year % 400 == 0) {
```

Comment:
Можно записать в одну строчку, если воспользоваться операцией ИЛИ
```suggestion
        return (year % 4 == 0 && year % 100 != 0) || year % 400 == 0;
```